Tensile/KernelWriterAssembly.py

```diff
@@ -1900,11 +1912,55 @@ def defineMACs(self, kernel, m, innerUnroll):
 
     # bfloat16
     elif kernel["ProblemType"]["DataType"].isBFloat16():
-      if self.version == (9,0,8):
-        if kernel["ProblemType"]["HighPrecisionAccumulate"]:
-          kStr += ""
-        else:
-          kStr += ""
+      if self.version == (9,0,8) and kernel["ProblemType"]["HighPrecisionAccumulate"]:
```

Comment:
Should be an architecture capability flag such as HasEccHalf.  See #744.

Reply:
Hi @sdquiring ,

I follow previous method to check should we use bfloat16 generator.
May you help to suggest how to modify??
What should we do for other code segment with self.version == (x, x, x) ???